{"Modified":"2020-07-22T20:02:00","Published":"2020-07-20T14:15:00","access":{"authentication":"SINGLE","complexity":"LOW","vector":"NETWORK"},"assigner":"cve@mitre.org","capec":[],"cvss":4,"cvss-time":"2020-07-22T20:02:00","cvss-vector":"AV:N/AC:L/Au:S/C:N/I:N/A:P","cwe":"NVD-CWE-noinfo","id":"CVE-2020-4466","impact":{"availability":"PARTIAL","confidentiality":"NONE","integrity":"NONE"},"last-modified":"2020-07-22T20:02:00","references":["https://exchange.xforce.ibmcloud.com/vulnerabilities/181563","https://www.ibm.com/support/pages/node/6250473"],"refmap":{"confirm":["https://www.ibm.com/support/pages/node/6250473"],"xf":["ibm-mq-cve20204466-dos (181563)"]},"summary":"IBM MQ for HPE NonStop 8.0.4 and 8.1.0 could allow a remote authenticated attacker could cause a denial of service due to an error within the Queue processing function. IBM X-Force ID: 181563.","vulnerable_configuration":[{"id":"cpe:2.3:a:ibm:mq_for_hpe_nonstop:8.0.4:*:*:*:*:*:*:*","title":"cpe:2.3:a:ibm:mq_for_hpe_nonstop:8.0.4:*:*:*:*:*:*:*"},{"id":"cpe:2.3:a:ibm:mq_for_hpe_nonstop:8.1.0:*:*:*:*:*:*:*","title":"cpe:2.3:a:ibm:mq_for_hpe_nonstop:8.1.0:*:*:*:*:*:*:*"}],"vulnerable_configuration_cpe_2_2":[],"vulnerable_product":["cpe:2.3:a:ibm:mq_for_hpe_nonstop:8.0.4:*:*:*:*:*:*:*","cpe:2.3:a:ibm:mq_for_hpe_nonstop:8.1.0:*:*:*:*:*:*:*"]}